### ISSUE TITLE
Clients can not login

### ISSUE THREAD

**Issue by OrlandoAmador** [osTicket/osTicket-1.7#317]
When ever one of my clients go to the login.php or tickets.php and enter their email and ticket number. all the page does is refresh. (the url does change to http://www.mysite.com/support/tickets.php?id=113071) Their is no error message at all. Now if the client uses link that was in the email it does let them in.

**Comment by fabiorauber** [comment]
Noticed the exact same problem.

**Comment by OrlandoAmador** [comment]
Ok I was able to find a solution. on https://github.com/protich/osTicket-1.7/commit/f3fccbc9516e7b0b3f8aa9099f6d96e768693df6#L0L137 I code that login.php and class.client.php and now it let you login. I did notice that 1 out of 10 times it just redirectt me to the main screen so it does not work 100% but that can also be due to other customization i have done. maybe it can lead to a better solution for now.
